platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -14,6 +14,8 @@ We are excited to announce Gradle @version@ (released [@releaseDate@](https://gr
 
 This release features [1](), [2](), ... [n](), and more.
 
+Gradle 9.0 has many bug fixes and other general improvements. As a major version, this release also has changes to deprecated APIs and behavior. Consult the [Gradle 8.x upgrade guide](userguide/upgrading_version_8.html) for guidance on removed APIs and behavior.
```

Comment:
🤔 This sounds like boilerplate, but would it be better to tweak it to downplay the "general improvements" and focus on the pruning of deprecated APIs?

Reply:
This one is just a note/reminder about verbiage from the Gradle 8.0 major release. This will not be in the final release as is.